src/components/StatusBadge/StatusBadge.stories.tsx

```diff
@@ -1,25 +1,28 @@
 import * as React from "react";
+
+import StatusBadge from "./StatusBadge";
 import bem from "../../utils/bem";
 import { withDesign } from "storybook-addon-designs";
 import { select, text } from "@storybook/addon-knobs";
 
 export default {
     title: "Status Badge",
+    component: StatusBadge,
     decorators: [withDesign],
 };
 
-export const StatusBadge = () => (
-    <div
-        className={bem("status-badge", [
-            select("Status of Badge", ["inactive", "warning"], "warning"),
-        ])}
-    >
+export const statusBadge = () => (
+    <StatusBadge level={select(
+        "Status of Badge",
+        ["low", "medium", "high"],
+        "low"
+    )}>
         {text("Status Badge Text", "Location is temporarily closed")}
-    </div>
+    </StatusBadge>
 );
 
-StatusBadge.storyName = "Status Badge";
-StatusBadge.parameters = {
+statusBadge.storyName = "Status Badge";
+statusBadge.parameters = {
     design: {
         type: "figma",
         url:
```

Comment:
Can you swap this URL out with `https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=11256%3A0`? Thanks!